### ISSUE TITLE
Low Prediction capacity

### ISSUE THREAD

**Issue by MarcosPalacios** [MarcosPalacios/Recurrent-Neural-Networks-for-Electricity-Price-Prediction#1]
Los modelos tienen poca capacidad predictiva.

He probado quitando variables para simplificar el modelo. La capacidad predictiva mejora un poco pero el overfitting es brutal y las curvas de val_loss y loss no convergen.

He probado aumentar los datos de training para reducir overfitting pero aun así la capacidad predictiva es mala.

También he probado cambiar la prediccion de 168 hs + 24hs a 24hs + 24hs, pero el modelo sigue sin funcionar.
